catalog/s3-policies.yaml

```diff
@@ -16,29 +15,68 @@
   resources:
     - "*"
 
-
 - sid: "DenyS3IncorrectEncryptionHeader"
   effect: "Deny"
   actions:
     - "s3:PutObject"
-  resources:
-    - "*"
   condition:
     - test: "StringNotEquals"
       variable: "s3:x-amz-server-side-encryption"
       values:
         - "AES256"
         - "aws:kms"
-
+  resources:
+    - "*"
 
 - sid: "DenyS3UnEncryptedObjectUploads"
   effect: "Deny"
   actions:
     - "s3:PutObject"
-  resources:
-    - "*"
   condition:
     - test: "Null"
       variable: "s3:x-amz-server-side-encryption"
       values:
         - true
+  resources:
+    - "*"
+
+- sid: "DenyNoTLSRequests"
+  effect: "Deny"
+  actions:
+    - "s3:*"
+  condition:
+    - test: "Bool"
+      variable: "aws:SecureTransport"
+      values:
+        - false
+  resources:
+    - "arn:aws:s3:::*/*"
+
+- sid: "RequireMFAS3Delete"
+  effect: "Deny"
+  actions:
+    - "s3:DeleteObject"
+  condition:
+    - test: "BoolIfExists"
+      variable: "aws:MultiFactorAuthPresent"
+      values:
+        - false
+  resources:
+    - "arn:aws:s3:::*/*"
+
+- sid: "S3RegionLockdown"
```

Comment:
At first glance it's hard to tell what this policy is supposed to do and the name isn't very descriptive. Consider adding a comment above with the policy's purpose.